jsonnet/telemeter/rules.libsonnet

```
@@ -130,6 +130,23 @@
                 topk(500, sum (acm_managed_cluster_info) by (managed_cluster_id, cloud, created_via, endpoint, instance, job, namespace, pod, service, vendor, version))
               |||,
             },
+            {
+              // OpenShift Cluster CPU-hours for the last hour
+              // The divisor of scalar(count_over_time(vector(1)[1h:5m])) allows us to get an effective average value having "absent data" treated as 0.
+              // max(...) by (_id) is used ensure a single datapoint per cluster ID
+              record: 'cluster:usage:workload:capacity_physical_cpu_hours',
+              expr: |||
+                max(sum_over_time(cluster:usage:workload:capacity_physical_cpu_cores:max:5m[1h:5m]) / scalar(count_over_time(vector(1)[1h:5m]))) by (_id)
```

scalar(count_over_time(vector(1)[1h:5m])) will always return 12 so you could simplify the expression.

Personal preference: "max by(...) (...)" reads better than "max(...) by(...)"

```suggestion
                max by (_id) (sum_over_time(cluster:usage:workload:capacity_physical_cpu_cores:max:5m[1h:5m]) / 12)
```

---

I actually like `max by` better too, now that I've seen it :-). I went ahead and made that change, I am looking forward to your thoughts on `12` vs. `13` vs. `scalar(count_over_time(vector(1)[1h:5m])))`. See above thread please.